### ISSUE TITLE
Wrap template in a <div> rather than a <form>?

### ISSUE THREAD

**Issue by mcblum** [vitalets/angular-xeditable#407]
Would this be possible? I changed it in the template but then there was an error with the controller. 

Thanks!
Matt

**Comment by lalitkapoor** [comment]
@mcblum yes you can do this, I had to make some changes to the `editableForm` directive. I'm trying to open a pull request for this actually because it should also solve https://github.com/vitalets/angular-xeditable/issues/157. My problem is I'm not sure how to run the tests to see if I broke anything else - https://github.com/vitalets/angular-xeditable/issues/405

**Comment by ckosloski** [comment]
@lalitkapoor are you going to create a pull request?
